www/pages/blog/state-of-greenwood-2022.md

```diff
@@ -0,0 +1,188 @@
+      :root {
+        --color-primary: #135;
+        --color-secondary: #74b238;
+        --font-family: 'Optima', sans-serif;
+        --font-size: 2rem;
+        --backgroundUrl: url('../assets/background.jpg');
+        width: 99%;
+        margin: auto;
+      }
+
+      :root h1 {
+        background-color: var(--color-secondary);
+      }
+
+      :root p {
+        color: var(--color-primary);
+        padding: 0 2rem;
+      }
+
+      :root img {
+        float: left;
+      }
+    </style>
+  </head>
+
+  <body>
+    <main>
+      <content-outlet></content-outlet>
+    </main>
+
+    <script> type="module">
+      // JavaScript can go here too!
+    </script>
+  </body>
+
+</html>
+```
+
+For a user of a theme pack, it would only require setting the `template` in a markdown file's frontmatter that matches the template name and presto!  Instant theming.  💯
+```md
+---
+template: title-card
+---
+
+# My Slide Title
+
+<style>
+  /* Overrides and customizations are super easy when it's just HTML and CSS! */
+  :root {
+    --color-primary: red;
+    --font-family: 'Comic Sans', sans-serif;
+    --backgroundUrl: url('../assets/rick-roll.gif');
+  }
+</style>
+
+This is my own slide content!
+
+![my-image](/assets/my-image.png)
+```
+
+![theme-pack](/assets/greenwood-starter-presentation.png)
+
+> _You can see [this example](https://github.com/thescientist13/knowing-your-tco) of the end user experience of a theme pack I used for a presentation I gave and our [guide on Theme Packs](/guides/theme-packs/) can help you learn more._
+
+### HTML Includes
+
+Last year we created a [**custom plugin**](https://github.com/ProjectEvergreen/greenwood/tree/master/packages/plugin-include-html) that aims to follow in the spirit of the [abandoned HTML Imports spec](https://www.html5rocks.com/en/tutorials/webcomponents/imports/) that was originally part of the initial Web Components "feature suite".  We thought we could breath a little life back into it for the benefit of Greenwood users.  Let's take a quick peak at what the HTML flavor of this API looks like, where you have static HTML that you want to reuse across your pages, like a global header or footer.
+
+So given a snippet of HTML
+```html
+<!-- src/includes/header.html -->
+<header class="my-include">
+  <h1>Welcome to my website!<h1>
+</header>
+```
+
+And a page template, you could then add this `<link>` tag
+```html
+<html>
+
+  <body>
+    <!-- rel and href attributes would be required -->
+    <link rel="html" href="/includes/header.html"></link>
+
+    <h2>Hello 👋</h2>
+
+  </body>
+
+<html>
+```
+
+And Greenwood will statically generate this
+```html
+<html>
+
+  <body>
+    <header class="my-include">
+      <h1>Welcome to my website!<h1>
+    </header>
+
+    <h2>Hello 👋</h2>
+
+  </body>
+
+<html>
+```
+
+> _Check out the docs on how to [use both options](https://github.com/ProjectEvergreen/greenwood/blob/master/packages/plugin-include-html/README.md) with this plugin._
+
+
+### HUD UI
```

Comment:
I think I might swap this one with _Interpolate Frontmatter_ and put _HUD UI_ in the "honorable mentions" section.